### ISSUE TITLE
[Bug] /ftbdim visit command related!

### ISSUE THREAD

**Issue by ixDarkLorD** [FTBTeam/FTB-Modpack-Issues#2932]
### Modpack

FTB Skies

### Modpack version

1.0.0

### Describe the bug

I don't know if this is supposed to work like that, but...
The `/ftbdim visit` command doesn't appear for the players without permission.

### Steps to reproduce

1. Go to the chat and type /ftbdim visit
2. Try to execute without having the op permission.

### Expected behaviour

-

**Comment by JustAnotherGhostName** [comment]
Im pretty sure that is as expected, if you dont have the permission node for something in minecraft, you shouldn't be able to know that command exists. However, I can say that when you add the permission node " command.ftbteamdimensions.visit: true" in ftb ranks, it didnt seem to work for me unless the player was opped.

**Comment by desht** [comment]
Yes, this is intended behaviour: `/ftbdim visit` is an admin command. But it should work with that Ranks permission node, so I'l look into that.

**Comment by desht** [comment]
Update: looked into it, it works.  I added `command.ftbteamdimensions.visit: true` to my rank, and was able to use the command:

```
> ftbranks node list member 
[08:43:12] [Server thread/INFO] [minecraft/MinecraftServer]: 1 permission node(s) in rank 'member':
[08:43:12] [Server thread/INFO] [minecraft/MinecraftServer]: --------------------
[08:43:12] [Server thread/INFO] [minecraft/MinecraftServer]: command.ftbteamdimensions.visit = true
[08:43:12] [Server thread/INFO] [minecraft/MinecraftServer]: --------------------
```
![image](https://user-images.githubusercontent.com/736831/230309312-94aa7dd3-ab65-457d-a1a1-81fa30a48544.png)

Definitely not op'd when I tested this.
